aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -549,6 +549,8 @@ Resources:
 <%=  component 'database'%>
 <% end -%>
 
+<%= resource_component 'codeprojects_resources' %>
```

Comment:
Should we only provision this when `cdn_enabled `?

Reply:
The current wildcard cert is applied to:

* Cloudfront [ENF08PEY9FX49](https://us-east-1.console.aws.amazon.com/cloudfront/v3/home?region=us-east-1#/distributions/ENF08PEY9FX49) - (static.codeprojects.org S3 Bucket)
* Cloudfront [EYAXK4MFKU8SN](https://us-east-1.console.aws.amazon.com/cloudfront/v3/home?region=us-east-1#/distributions/EYAXK4MFKU8SN) - (bramble-download.codeprojects.org S3 Bucket)
* ELB [sites-codeprojects-org](https://us-east-1.console.aws.amazon.com/ec2/v2/home?region=us-east-1#LoadBalancers:search=e846f2c8251511a5;sort=loadBalancerName)
* ELB [staging-codeprojects-org](https://us-east-1.console.aws.amazon.com/ec2/v2/home?region=us-east-1#LoadBalancers:search=df943227dca985d8;sort=loadBalancerName)
* ELB [test-codeprojects-org](https://us-east-1.console.aws.amazon.com/ec2/v2/home?region=us-east-1#LoadBalancers:search=e9ae226fd00e4232;sort=loadBalancerName)

We want this in all three environments regardless of CDN. One next step here would be to get the load balancers into this template component too.